frontend/features/page/sections/ContactsSection/ContactsSection.tsx

```diff
@@ -0,0 +1,567 @@
+                });
+              }}
+              value={state.name}
+              id="1"
+              borderX="none"
+              borderTop="none"
+              borderRadius="0"
+              pos="relative"
+              outline="none"
+              borderBottom="1px solid rgb(148, 148, 148)"
+              w="full"
+              minH="10"
+              py="2.5"
+              px="0"
+              boxSizing="border-box"
+              fontSize="sm"
+              resize="none"
+              display="inline-block"
+              lineHeight="1.4em"
+              _focus={{
+                "& ~ span": {
+                  width: "100%",
+                },
+              }}
+              type="text"
+              placeholder="Peter Smith"
+              isRequired
+            />
+            <FormErrorMessage>{"Name is required"}</FormErrorMessage>
+            <Box
+              as="span"
+              pos="absolute"
+              bottom="0"
+              left="0"
+              w="0"
+              h="1px"
+              backgroundColor="dark"
+              transition="all 0.4s ease 0s"></Box>
+          </FormControl>
+        </Box>
+        <Box
+          w={{
+            base: "calc(100% - 20px)",
+            sm: "calc(50% - 20px)",
+            md: "calc(100% - 20px)",
+            lg: "calc(50% - 20px)",
+          }}
+          m="2.5"
+          pos="relative"
+          display="inline-block">
+          <FormControl id="email" isInvalid={state.email === "" ? true : false}>
+            <FormLabel
+              mb="0"
+              fontWeight="400"
+              fontSize="sm"
+              textTransform="uppercase"
+              letterSpacing="0.1em"
+              pos="relative"
+              w="full"
+              display="block">
+              Email
+            </FormLabel>
+            <Input
+              onChange={(event) => {
+                dispatch({
+                  type: FormActionType.UpdateField,
+                  value: event.target.value,
+                  fieldName: FormFieldName.Email,
+                });
+              }}
+              id="2"
+              value={state.email}
+              isRequired
+              borderX="none"
+              borderTop="none"
+              borderRadius="0"
+              pos="relative"
+              outline="none"
+              borderBottom="1px solid rgb(148, 148, 148)"
+              w="full"
+              minH="10"
+              py="2.5"
+              px="0"
+              boxSizing="border-box"
+              fontSize="sm"
+              resize="none"
+              display="block"
+              lineHeight="1.4em"
+              _focus={{
+                "& ~ span": {
+                  width: "100%",
+                },
+              }}
+              type="email"
+              placeholder="example@yourdomain.com"
+            />
+            <FormErrorMessage>{"Email is required"}</FormErrorMessage>
```

Comment:
Instead of this `<FormErrorMessage>{"Email is required"}</FormErrorMessage>` you can just write this `<FormErrorMessage>Email is required</FormErrorMessage>`. The thing here is that email is not just required, it should also be valid. So here you are going to display an error message based on the error that the user has made.. You probably want to expand your form state to account also for errors, e.g.

```ts
interface FormState {
  values: {
    name: string;
    ...
  },
  errors: {
    name: string;
    ...
  }
}
```